packages/bitcore-client/bin/wallet-import

```diff
@@ -7,6 +7,7 @@ const Wallet = require('../lib/wallet');
 program
   .version(require('../package.json').version)
   .option('--path <path>', 'REQUIRED - Where wallet is stored')
+  .option('--password <password>', 'REQUIRED - password to unlock the wallet so the private keys can be encrypted')
```

Comment:
we need to prompt for the password, don't want that in your bash history ;)

Reply:
Makes sense! Other than that looks good? I have another coming soon with the level dB storage and retrieval working